### ISSUE TITLE
Can't seemingly specify project file using command line

### ISSUE THREAD

**Issue by dukc** [bridgedotnet/Bridge#3371]
I am new to using Bridge. I do not want to use Visual Studio, so I decided to install it using NuGet, which went smoothly. But when it's time to use it, I hit the wall. About the syntax, it says:
`Usage: bridge.exe [options] (<project-file>|<assembly-file>)
       bridge.exe [-h|--help]
`
But apparently I can't find how it excepts the project file to be specified. I was unable to find any example which would claify what I'm doing wrong

### Steps To Reproduce

1. Install Bridge using NuGet.
2. Put it into your PATH enviroment variable.
3. Make a .cs file you want to combine and a projec file, "build.xml" in this case.
4. Attempt to compile:
`bridge build.xml`
or `bridge (build.xml)`

If it matters, I'm using windows 10.

### Expected Result

A succesful compilation, or error messages about the code.

### Actual Result
`Invalid argument: build.xml
Usage: bridge.exe [options] (<project-file>|<assembly-file>)
       bridge.exe [-h|--help]

-h --help                  This help message.
-c --configuration <name>  Configuration name (Debug/Release etc)
                           [default: none].
-P --platform <name>       Platform name (AnyCPU etc) [default: none].
-S --settings <name:value> Comma-delimited list of project settings
                           I.e -S name1:value1,name2:value2)
                           List of allowed settings:
                             AssemblyName, CheckForOverflowUnderflow,
                             Configuration, DefineConstants,
                             OutputPath, OutDir, OutputType,
                             Platform, RootNamespace
                           options -c, -P and -D have priority over -S
-r --rebuild               Force assembly rebuilding.
--nocore                   Do not extract core javascript files.
-D --define <const-list>   Semicolon-delimited list of project constants.
-b --bridge <file>         Bridge.dll file location (currently unused).
-s --source <file>         Source files name/pattern [default: *.cs].
-f --folder <path>         Builder working directory relative to current WD
                           [default: current wd].
-R --recursive             Recursively search for .cs source files inside
                           current workind directory.
-notimestamp --notimestamp Do not show timestamp in log messages
                           [default: shows timestamp]
`
or 
`Invalid argument: build.xml
...`

**Comment by geoffreymcgill** [comment]
You're going to want to use the new **Bridge CLI**, which should be linked on the [bridge.net/download](https://bridge.net/download) page later today. If you want to get started now, clone the following project, then build in **Release** mode.

https://github.com/bridgedotnet/CLI

Then double-click the bin generated **bridge-cli.exe** file to install.

**Comment by dukc** [comment]
Thanks, I think I try. But shouldn't it work with ordinary cmd too?

**Comment by fabriciomurta** [comment]
I don't think bridge.exe supports the "build.xml" file format, although you didn't show it. You can point to csproj, sln or individual .cs files. Or an already-built assembly of a bridge project.

**Comment by dukc** [comment]
Nope. 'bridge app.cs' gave the same result. It is, at least apparently, an argument parsing problem, not a file type problem. But now when the CLI is in distribution and works for me I don't personally care anymore.

(I did also try building CLI myself but ran into another problem with package manager which appeared to be from Visual Studio part ...yet another reason for me to avoid depending on it!)

**Comment by geoffreymcgill** [comment]
> I did also try building CLI myself but ran into another problem with package manager which appeared to be from Visual Studio part

@dukc – Could you provide some more details on that comment? Any error messages? 

If there's an issue with the build process, we would like to isolate the problem and fix.

**Comment by geoffreymcgill** [comment]
Can you update your local repo with the latest from GitHub, then try to rebuild the project. The request to 16.7.0 should not have been added to **master**, but it was updated to about 3 days ago to 16.6.61.

https://github.com/bridgedotnet/CLI/commit/079ab5a81122a9f0fa2b0bb0957b95b7067d6c05

**Comment by dukc** [comment]
Thanks but no thanks, since there's the distribution build already.
